mscore/scoreview.cpp

```diff
@@ -4727,6 +4727,12 @@ QList<Element*> ScoreView::elementsNear(QPointF p)
       QRectF r(p.x() - w, p.y() - w, 3.0 * w, 3.0 * w);
 
       QList<Element*> el = page->items(r);
+      for(int i = 0; i < 3; i++) {
```

Comment:
I think here you need to add a comment what is 3  
I think it's even better to add a constant (`mscore.h`), since it is used in several places. 
The name of the constant will show what it is and where it is used.